Device/Device/schema.json

```diff
@@ -0,0 +1,199 @@
+            "type": "StructuredValue",
+            "metadata": {
+                "dateModified": {
+                    "descr": "It captures the last modification timestamp of this attribute.",
+                    "type": "DateTime"
+                }
+            }
+        },
+        "location": {
+            "descr": "Location of this device represented by a GeoJSON geometry of type point.",
+            "type": "geo:json",
+            "allowedvalues": {
+                "normative": "https://tools.ietf.org/html/rfc7946"
+            }
+        },
+        "name": {
+            "descr": "A mnemonic name given to the device.",
+            "type": "name",
+            "allowedvalues": {
+                "normative": "https://schema.org/name"
+            }
+        },
+        "description": {
+            "descr": "Device's description.",
+            "type": "description",
+            "allowedvalues": {
+                "normative": "https://schema.org/description"
+            }
+        },
+        "dateInstalled": {
+            "descr": "A timestamp which denotes when the device was installed (if it requires installation).",
+            "type": "DateTime"
+        },
+        "dateFirstUsed": {
+            "descr": "A timestamp which denotes when the device was first used.",
+            "type": "DateTime"
+        },
+        "dateManufactured": {
+            "descr": "A timestamp which denotes when the device was manufactured.",
+            "type": "DateTime"
+        },
+        "hardwareVersion": {
+            "descr": "The hardware version of this device.",
+            "type": "Text"
+        },
+        "hardwareVersion": {
+            "descr": "The software version of this device.",
+            "type": "Text"
+        },
+        "firmwareVersion": {
+            "descr": "The firmware version of this device.",
+            "type": "Text"
+        },
+        "osVersion": {
+            "descr": "The version of the host operating system device.",
+            "type": "Text"
+        },
+        "dateLastCalibration": {
+            "descr": "A timestamp which denotes when the last calibration of the device happened.",
+            "type": "DateTime"
+        },
+        "serialNumber": {
+            "descr": "The serial number assigned by the manufacturer.",
+            "type": "serialNumber",
+            "allowedvalues": {
+                "normative": "https://schema.org/serialNumber"
+            }
+        },
+        "provider": {
+            "descr": "The provider of the device.",
+            "type": "provider",
+            "allowedvalues": {
+                "normative": "https://schema.org/provider"
```

Comment:
I agree with that, but how do will we represent normative reference? 
The machine can't use that because a normative can be any schema, documentation, RF, in any format. 
But it can be used to advise to user/developer which value should be used for this attribute.
Maybe:
```
"provider": {
            "description": "The provider of the device.",
            "type": "provider",
             "normative": "https://schema.org/provider"
        },
```